make/source.mk

```diff
@@ -359,7 +361,9 @@ SIZE_OPTIMISED_SRC := $(SIZE_OPTIMISED_SRC) \
             osd/osd.c \
             osd/osd_elements.c \
             osd/osd_warnings.c \
-            rx/rx_bind.c
+            rx/rx_bind.c \
+            io/vtx_msp.c \
+            cms/cms_menu_vtx_msp.c \
```

Comment:
Additional `\` needs one more line to be empty. Jus to let you decide if we want that.

Reply:
removed